test/blackbox-tests/test-cases/pkg/depexts.t

```diff
@@ -0,0 +1,104 @@
+Exercises end to end locking and building a simple project.
```

Comment:
Can you split this test in two parts

1. One that demonstrates how we convert the `depext` field in opam to lock directory packages. This test should use the solver.
2. One that demonstrates how this new field is affecting the error messages. This test should not use the solver.

Reply:
Done.